lib-ce/emqx_modules/src/emqx_mod_delayed.erl

```diff
@@ -163,8 +169,9 @@ handle_info(Info, State) ->
     ?LOG(error, "Unexpected info: ~p", [Info]),
     {noreply, State}.
 
-terminate(_Reason, #{timer := TRef}) ->
-    emqx_misc:cancel_timer(TRef).
+terminate(_Reason, #{timer := PublishTimer, stats_timer := StatsTimer}) ->
```

Comment:
Backward compatibility: the stats timer is newly added, may not exist after relup